modules/gapi/test/common/gapi_tests_common.hpp

```diff
@@ -155,12 +155,58 @@ class TestFunctional
         }
     }
 
+    void initMatsFromImages(int channels, const std::string& fileNamePattern, int format)
+    {
+        initTestDataPath();
+        switch (channels)
+        {
+            case 1:
+                in_mat1 = cv::imread(perf::TestBase::getDataPath(cv::format(fileNamePattern.c_str(),
```

Comment:
sometimes `using` really helps

Reply:
_lambda_ used, I hope it looks good enough